.travis.yml

```diff
@@ -8,7 +8,7 @@ matrix:
     allow_failures:
         - python: 3.6
 install:
-    - pip install -r requirements.txt
+    # - pip install -r requirements.txt
```

Comment:
@cclauss not sure how travis works internally, but doesn't it set up a local virtualenv? if you remove the requirements installation part isn't it going to fail always?

Reply:
* __requirements.txt__ is commented out because this repo does not have a file by that name.
* The __before_script__ does not need the external modules...  It is merely doing analysis of the code in this repo.
* The __script__ piece is currently just a placeholder until some pytests or similar are written.